src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariantsIntegrationTest.java

```diff
@@ -289,6 +289,21 @@ public void testVariantTypeSelection() throws IOException {
         spec.executeTest("testVariantTypeSelection--" + testFile, this);
     }
 
+    /**
+     * Test including variant types in GVCF files.
+     */
+    @Test
+    public void testVariantTypeSelectionForGVCF() throws IOException {
+        final String testFile = getToolTestDataDir() + "gvcfExample.g.vcf";
+
+        final IntegrationTestSpec spec = new IntegrationTestSpec(
+                baseTestString(" --select-type-to-include SNP --ignore-non-ref-in-types ",testFile),
+                Collections.singletonList(getToolTestDataDir() + "expected/" + "testSelectVariants_VariantTypeSelectionForGVCF.vcf")
```

Comment:
We've been trying to move away from brittle exact match integration tests, but I think that this is fine here since there are no opportunities for numerical changes.